src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    
+    transition: transform 300ms, color 300ms;
+
+    &:hover {
```

Comment:
It's small project but it would be better to keep colors and constants of animation in variables.

Reply:
Thank you for your suggestions. I'll fix it